include/nbl/asset/IAssetManager.h

```diff
@@ -322,7 +320,7 @@ class IAssetManager : public core::IReferenceCounted, public core::QuitSignallin
 
             std::string filePath = _filePath;
             _override->getLoadFilename(filePath, ctx, _hierarchyLevel);
-            io::IReadFile* file = m_fileSystem->createAndOpenFile(filePath.c_str());
+            system::IFile* file = m_system->createAndOpenFile(filePath.c_str());
```

Comment:
I guess you just did find&replace here, but note that theres no `createAndOpenFile` in ISystem + you wont have to do `.c_str()` any more

Reply:
Yeah i actually added `createAndOpenFile` to `ISystem`. but yeah, that `.c_str()` is unnecessary

Reply:
> Yeah i actually added createAndOpenFile to ISystem

yea, saw that later, get rid of it